app/src/main/java/android/support/v4/widget/FixedDrawerLayout.java

```diff
@@ -24,11 +24,11 @@ public FixedDrawerLayout(Context context, AttributeSet attrs, int defStyle) {
         super(context, attrs, defStyle);
     }
 
-    @Override
-    boolean isContentView(View child) {
-        if (child == null) {
-            return false;
-        }
-        return super.isContentView(child);
-    }
+//    @Override
```

Comment:
why commented out ?

Reply:
This override only works on android support. Will just delete it.